overlord/snapshotstate/backend/backend.go

```diff
@@ -315,17 +316,24 @@ func Save(ctx context.Context, id uint64, si *snap.Info, cfg map[string]interfac
 		Version:  si.Version,
 		Epoch:    si.Epoch,
 		Time:     timeNow(),
+		Options:  dynSnapshotOpts, // Pass only dynamic snapshot options here
```

Comment:
This comment could explain _why_ we only care about dynamic snapshot options

Reply:
Added explanation.